Mathlib/MeasureTheory/Function/LpSeminorm/Basic.lean

```diff
@@ -114,8 +114,6 @@ theorem memℒp_def {α} {_ : MeasurableSpace α} (f : α → E) (p : ℝ≥0∞
     Memℒp f p μ ↔ (AEStronglyMeasurable f μ ∧ snorm f p μ < ∞) :=
   Iff.rfl
 
-attribute [eqns memℒp_def] Memℒp
```

Comment:
The porting note above (leanprover/lean4#2243) suggests that maybe the `_def` lemma itself can be deleted?